lang/python/python-pycares/Makefile

```diff
@@ -0,0 +1,48 @@
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python-pycares
+PKG_VERSION:=4.2.2
+PKG_RELEASE:=1
+
+PYPI_NAME:=pycares
+PKG_HASH:=e1f57a8004370080694bd6fb969a1ffc9171a59c6824d54f791c1b2e4d298385
+
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE
+PKG_MAINTAINER:=Fabian Lipken <dynasticorpheus@gmail.com>
+
+PKG_BUILD_DEPENDS:=libffi/host
+
+HOST_PYTHON3_PACKAGE_BUILD_DEPENDS:=cffi  # cffi>=1.5.0
+
+include ../pypi.mk
+include $(INCLUDE_DIR)/package.mk
+include ../python3-package.mk
+
+define Package/python3-pycares
+  SECTION:=lang
+  CATEGORY:=Languages
+  SUBMENU:=Python
+  TITLE:=Python interface for c-ares
+  URL:=https://github.com/saghul/pycares
+  DEPENDS:=+python3-light
+endef
+
+define Package/python3-pycares-src
+  SECTION:=lang
+  CATEGORY:=Languages
+  SUBMENU:=Python
+  TITLE:=Python interface for c-ares
+  URL:=https://github.com/saghul/pycares
+  DEPENDS:=+python3-light
+endef
+
+define Package/python3-pycares/description
+Python interface for c-ares
+endef
+
+$(eval $(call Py3Package,python3-pycares))
+$(eval $(call BuildPackage,python3-pycares))
+$(eval $(call BuildPackage,python3-pycares-src))
+
```

Comment:
since i'm complaining anyway: you can remove this empty last-line; but it's more of a nitpick (or personal feeling)

feel free to disregard it;